andela/activo-web
# #158863118 Fix Navbar responsiveness

**What does this PR do?**

It fixes the responsiveness of the navigation bar and the asset pane on large screen

**Description of Task to be completed?**
   - fix navbar responsiveness
   - centralize the content of the settings page
    - fix overflow on the x-axis of the webpage
    - add min-height to the dashboard container

**How should this be manually tested?**
 - `yarn start:dev` to open the web browser on `port 3000` 
- On small devices change your display options to `scaled` with more space to make sure it's responsive or view the webpage on a desktop

**Any background context you want to provide?**

N/A

**What are the relevant pivotal tracker stories?**

[#158863118](https://www.pivotaltracker.com/story/show/58863118)

**Screenshots (if appropriate)**
Default display for laptops:
<img width="1436" alt="screen shot 2018-07-11 at 3 12 27 pm" src="https://user-images.githubusercontent.com/29452947/42577717-e39ae06a-851c-11e8-8c0d-373fad7465d8.png">

Scaled display with more space or desktop:
<img width="1680" alt="screen shot 2018-07-11 at 3 15 27 pm" src="https://user-images.githubusercontent.com/29452947/42577893-48a95a04-851d-11e8-8e0b-04bdf4af4a1e.png">



**Questions:**

N/A
